subgraphs/rocket-pool/src/Constants/constants.ts

```diff
@@ -0,0 +1,196 @@
+import { BigDecimal, BigInt, Bytes } from "@graphprotocol/graph-ts";
+
+////////////////////
+///// Versions /////
+////////////////////
+
+export const PROTOCOL_NAME = "RocketPool";
+export const PROTOCOL_SLUG = "RocketPool";
+export const PROTOCOL_SCHEMA_VERSION = "1.3.0";
```

Comment:
Use the versioning logic steegecs created: https://github.com/messari/subgraphs/blob/master/subgraphs/aave-v2-forks/src/helpers.ts#L94-L96

Reply:
@dmelotik  @steegecs versions.ts is .gitignore 'd so I can't see what that logic is currently

Reply:
not sure what command you're referring to with messari b

Reply:
https://discord.com/channels/953684103012683796/953686506990608435/1035613688016674836

The `messari-subgraph-cli` package